Add FAQ about why we chose Alevin-fry (#21)
AlexsLemonade/scpca-docs: docs/faq.md

# Frequently Asked Questions 

> #### Why did we use Alevin-fry for processing? 

Reviewer: General comment: I'd pick either Cell Ranger `count` or `cellranger count` but don't mix and also consider how you're talking about Alevin-fry in comparison (no code blocks).

> We have found that Alevin-fry offers a faster, more memory efficient approach to processing single-cell and single-nuclei RNA-sequencing data than using 10X Genomics' [Cellranger count,](https://support.10xgenomics.com/single-cell-gene-expression/software/pipelines/latest/using/count) without sacrificing accuracy. 

Reviewer: ```suggestion
In our own benchmarking, we found that Alevin-fry offers a faster, more memory efficient approach to processing single-cell and single-nuclei RNA-sequencing data than 10X Genomics' [Cell Ranger count,](https://support.10xgenomics.com/single-cell-gene-expression/software/pipelines/latest/using/count) without sacrificing accuracy. 
```

> Alevin-fry uses approximately 12-16 GB of memory per sample and completes alignment and quantification in less than an hour, while `Cellranger count` uses up to 25-30 GB of memory per sample taking anywhere from 2-8 hours to align and quantify one sample.

Reviewer: ```suggestion
Alevin-fry uses approximately 12-16 GB of memory per sample and completes alignment and quantification in less than an hour, while `cellranger count` uses up to 25-30 GB of memory per sample taking anywhere from 2-8 hours to align and quantify one sample.
```

Additionally, comparison of the quantification output from samples processed with both Alevin-fry and Cell Ranger revealed similar distributions of UMI/cell and genes detected/cell among both tools.

![](../images/total_umi_per_cell.png)

> ![](../images/total_genes_per_cell.png)

Reviewer: Two for scRNA-seq and snRNA-seq seems sufficient to me but maybe we should consider linking to the `alsf-scpca` repository?

> We also compared the mean gene expression reported for each gene by both methods and observed a high correlation with a pearson R correlation coefficient of 0.98.  

Reviewer: Proper noun (may also impact dictionary):

```suggestion
We also compared the mean gene expression reported for each gene by both methods and observed a high correlation with a Pearson R correlation coefficient of 0.98.  
```

> *Insert Correlation graph comparing mean gene expression?*

Reviewer: That seems reasonable.

> Recent reports have supported our findings, showing that Alevin-fry can process single-cell and single-nuclei data much more quickly and efficiently then the next available methods, while also decreasing the false positive rate of gene detection, commonly seen in methods that utilize transcriptome alignment ([He _et al._ 2021](https://doi.org/10.1101/2021.06.29.450377)).

Reviewer: Changes for clarity:

```suggestion
Recent reports from others support our findings. 
[He _et al._ (2021)](https://doi.org/10.1101/2021.06.29.450377)) demonstrated that Alevin-fry can process single-cell and single-nuclei data more quickly and efficiently then other available methods, while also decreasing the false positive rate of gene detection that is commonly seen in methods that utilize transcriptome alignment.
```

> 

Reviewer: Maybe say something here (or in the first paragraph) about how picking something based on similarity to Cell Ranger results is part of an effort to make things comparable to pipelines we expect people run on their own data? You say this in the comment but not in the answer itself!

# Frequently Asked Questions 

#### Why did we use Alevin-fry for processing? 

> In an effort to allow users to easily compare data available on the portal to their own processed data, we aimed to process all of the data using a method that is comparable to widely used pipelines.

Reviewer: ```suggestion
We aimed to process all of the data in the portal such that it is comparable to widely used pipelines, namely Cell Ranger from 10x Genomics.
```

> In our own benchmarking, we have found that Alevin-fry offers a faster, more memory efficient approach to processing single-cell and single-nuclei RNA-sequencing data than the commonly utilized 10X Genomics' [Cell Ranger,](https://support.10xgenomics.com/single-cell-gene-expression/software/pipelines/latest/using/count) without sacrificing accuracy. 

Reviewer: ```suggestion
In our own benchmarking, we found that Alevin-fry offers a faster, more memory efficient approach to processing single-cell and single-nuclei RNA-sequencing data than [Cell Ranger](https://support.10xgenomics.com/single-cell-gene-expression/software/pipelines/latest/using/count) and produces highly similar results.
```

> Alevin-fry uses approximately 12-16 GB of memory per sample and completes alignment and quantification in less than an hour, while Cell Ranger uses up to 25-30 GB of memory per sample taking anywhere from 2-8 hours to align and quantify one sample.

Reviewer 2: I think it is important here to mention what mode we are using. Partly because if we used sketch mode we could get these numbers much lower, and I don't want to make people question the benchmarking difference they may see with other descriptions of alevin-fry.
```suggestion
In the configuration that we are using ("selective alignment" mapping to a human transcriptome that includes introns), Alevin-fry uses approximately 12-16 GB of memory per sample and completes mapping and quantification in less than an hour. 
By contrast, Cell Ranger uses up to 25-30 GB of memory per sample and takes anywhere from 2-8 hours to align and quantify one sample.
```

Additionally, comparison of the quantification output from samples processed with both Alevin-fry and Cell Ranger revealed similar distributions of UMI/cell and genes detected/cell among both tools.

![](https://github.com/AlexsLemonade/alsf-scpca/blob/dcebe62514b3a47a26b5923e7a1bdb1bb277e83b/analysis/docs-figures/plots/total_umi_per_cell.png?raw=true)

![](https://github.com/AlexsLemonade/alsf-scpca/blob/dcebe62514b3a47a26b5923e7a1bdb1bb277e83b/analysis/docs-figures/plots/total_genes_per_cell.png?raw=true)

We also compared the mean gene expression reported for each gene by both methods and observed a high correlation with a Pearson R correlation coefficient of 0.98.  

![](https://github.com/AlexsLemonade/alsf-scpca/blob/dcebe62514b3a47a26b5923e7a1bdb1bb277e83b/analysis/docs-figures/plots/gene_exp_correlation.png?raw=true)

Recent reports from others support our findings. 

> [He _et al._ (2021)](https://doi.org/10.1101/2021.06.29.450377)) demonstrated that Alevin-fry can process single-cell and single-nuclei data more quickly and efficiently then other available methods, while also decreasing the false positive rate of gene detection that is commonly seen in methods that utilize transcriptome alignment.

Author: Do we want to add another sentence at the end of this and incorporate these references then? Something like this? Or did you have another place in mind to put these references? 

> [You _et al._ (2021)](https://doi.org/10.1101/2021.06.17.448895) and [Tian _et al._ (2019)](https://doi.org/10.1038/s41592-019-0425-8) have also noted that results from different pre-processing workflows for single-cell RNA-sequencing analysis tend to result in compatible results downstream.

> [He _et al._ (2021)](https://doi.org/10.1101/2021.06.29.450377)) demonstrated that Alevin-fry can process single-cell and single-nuclei data more quickly and efficiently then other available methods, while also decreasing the false positive rate of gene detection that is commonly seen in methods that utilize transcriptome alignment.

Reviewer 2: A second sentence here seems good to me!
